lib/bitcoin/IBitcoinConfig.ts

```diff
@@ -16,4 +16,5 @@ export interface IBitcoinConfig {
   requestTimeoutInMilliseconds: number | undefined;
   requestMaxRetries: number | undefined;
   transactionPollPeriodInSeconds: number | undefined;
+  transactionFeeMarkupPercentage: number | undefined;
```

Comment:
1. We need a corresponding change in the ion repo to include this new parameter.
1. Can we keep this variable as required? And by default set it to 0?